libs/designsystem/src/lib/components/form-field/input/input.component.spec.ts

```diff
@@ -149,3 +149,17 @@ describe('InputComponent', () => {
     });
   });
 });
+
```

Comment:
Suggestions: Add a test that shows that kirby-input defaults to large size - hence a test showing there is no breaking changes.

Reply:
There is already a couple of tests that checks the default height, but as size is now a thing, the tests could probably be structured more logically. Will look into it 👍

Reply:
After consideration I decided to leave the test structure as-is. I just changed some wording to make it more explicit the default height is tested.